src/lists.py

```diff
@@ -50,11 +50,31 @@ def print_list(x):
         times_three = []
         print_list(times_three)
 
+
+# multiplying with base python
+y=[]
+for i in x:
+    y.append(3.i)
+    print(y)
+
+# another approach
+[3.i for i in x]
+
+
     case "even":
         # Put your solution to the third exercise here
         even = []
         print_list(even)
 
+y=[]
+for i in x:
+    if i %2 <=0:
+    y.append(3.i)
+    print(y)
+
+[i for i in x if i %2 <= 0]
```

Comment:
While it certainly doesn't matter for this particular exercise, you want `i%2 == 0` and not `i%2 <= 0`. The remainder of a division can easily be negative, so if someone sees the inequality, they will probably assume that something complicated is going on.  There isn't (and can't in this case because we take the remainder with respect to two), but it is still better to have easy-to-read code.